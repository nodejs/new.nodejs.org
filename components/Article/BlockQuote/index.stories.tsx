import BlockQuote from './index';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof BlockQuote>;
type Meta = MetaObj<typeof BlockQuote>;

export const Default: Story = {
  args: { children: 'This is a block quote' },
};

export const MultipleParagraph: Story = {
  args: {
    children: [
      <p key={1}>This is a block quote 1</p>,
      <p key={2}>This is a block quote 2</p>,
    ],
  },
};

export default { component: BlockQuote } as Meta;
