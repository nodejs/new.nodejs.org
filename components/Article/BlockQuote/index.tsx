import styles from './index.module.scss';
import type { FC, PropsWithChildren } from 'react';

const BlockQuote: FC<PropsWithChildren> = ({ children }) => (
  <div className={styles.blockQuote}>{children}</div>
);

export default BlockQuote;
