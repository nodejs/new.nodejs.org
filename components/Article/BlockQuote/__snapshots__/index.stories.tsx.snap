// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article/BlockQuote Default smoke-test 1`] = `
<div class="BlockQuote_blockQuote___32dx">
  This is a block quote
</div>
`;

exports[`Article/BlockQuote MultipleParagraph smoke-test 1`] = `
<div class="BlockQuote_blockQuote___32dx">
  <p>
    This is a block quote 1
  </p>
  <p>
    This is a block quote 2
  </p>
</div>
`;
